katello/katello/helper.rb

```diff
@@ -1,14 +1,17 @@
 module KatelloUtilities
   module Helper
     def last_scenario
-      last_scenario_yaml = '/etc/foreman-installer/scenarios.d/last_scenario.yaml'
-      if File.exists?(last_scenario_yaml)
+      if File.exist?(last_scenario_yaml)
```

Comment:
It looks like we mostly use ```File.exists``` and I think that's also easier to read

Reply:
I think so too - but this is what rubocop prefers. But if that's more consistent (we are overriding that cop elsewhere?) then I will change it

Reply:
Actually it looks like foreman proper uses `File.exist?` which is rubocop preferred as I mentioned. I'm leaning toward leaving it as-is.

Reply:
yes, whatever gets the script more prepared for rubocop has my vote

Reply:
I prefer as little change as possible since we should really get rid of all these files from the packaging repository. They were accepted on the premise that there would be an effort to move them to another repository. I'd also be against huge changes like https://github.com/theforeman/foreman-packaging/pull/2115

Reply:
You mentioned preferring as little change as possible, yet in another comment are asking me to change service restart commands which were simply copy/paste as I refactored the code - so I'm getting a mixed message. `File.exist?`seems most appropriate in terms of consistency and rubocop's needs. Let's keep it!

Reply:
You're right, I didn't realize that the restart code was already there and thought it was a new line. In that case I'm fine with keeping the current restart code.